test/tools/utils.js

```diff
@@ -1,8 +1,17 @@
 const {bufferToHex, toBuffer,setLengthLeft} = require('ethereumjs-util');
 
+const bytesToBytesN = (input, len) => {
```

Comment:
Function that converts an input (string|number|BN) to hex honoring the leading 0s. This is in theory fixed by `web3.utils.toHex` in newer version of web3